addons/web/static/src/js/views/form/form_controller.js

```diff
@@ -141,7 +141,11 @@ var FormController = BasicController.extend({
      * @param {jQuery} [$node]
      */
     renderButtons: function ($node) {
-        var $footer = this.footerToButtons ? this.renderer.$el && this.renderer.$('footer') : null;
+        var $footer = this.footerToButtons
+            ? this.renderer.$el && this.renderer.$('footer').filter(function () {
+                return !this.closest('.oe_form_field');
+            })
```

Comment:
Maybe extract this in a function to prevent code duplication.
Also, I'd use `o_field_widget` instead of `oe_form_field` (this is the generic classname for fields, instead of the one of field html). This shouldn't change anything, except remove a reference in web from something defined in web_editor.

Reply:
changed to use o_field_widget